README.rst

```diff
@@ -33,6 +33,118 @@ Features
+
+
+   # Define coordinate values for all dimensions of the data
+   coords={
+    "trial": range(100),
+    "features": ["sunlight hours", "water amount", "soil nitrogen"],
+   }
+
+   # Define generative model
+   with pm.Model(coords=coords) as generative_model:
+      x = pm.Data("x", x_data, dims=["trial", "features"])
+
+      # Model parameters
+      betas = pm.Normal("betas", dims="features")
+      sigma = pm.HalfNormal("sigma")
+
+      # Linear model
+      mu = x @ betas
+
+      # Likelihood
+      # Assuming we measure deviation of each plant from baseline
+      plant_growth = pm.Normal("plant growth", mu, sigma, dims="trial")
+
+
+   # Generating data from model by fixing parameters
+   fixed_parameters = {
+    "betas": [5, 20, 2],
+    "sigma": 0.5,
+   }
+   with pm.do(generative_model, fixed_parameters) as synthetic_model:
+      idata = pm.sample_prior_predictive(random_seed=seed) # Sample from prior predictive distribution.
+      synthetic_y = idata.prior["plant growth (z-scored)"].sel(draw=0, chain=0)
+
+
+   # Infer parameters conditioned on observed data
+   with pm.observe(generative_model, {"plant growth (z-scored)": synthetic_y}) as inference_model:
+    idata = pm.sample(random_seed=seed)
+
+   summary = pm.stats.summary(idata, var_names=["betas", "sigma"]))
+   print(summary)
+
+
+From the summary, we can see that the mean of the inferred parameters are very close to the fixed parameters
+
+=====================  ======  =====  ========  =========  ===========  =========  ==========  ==========  =======
+Params                  mean     sd    hdi_3%    hdi_97%    mcse_mean    mcse_sd    ess_bulk    ess_tail    r_hat
+=====================  ======  =====  ========  =========  ===========  =========  ==========  ==========  =======
+betas[sunlight hours]   4.972  0.054     4.866      5.066        0.001      0.001        3003        1257        1
+betas[water amount]    19.963  0.051    19.872     20.062        0.001      0.001        3112        1658        1
+betas[soil nitrogen]    1.994  0.055     1.899      2.107        0.001      0.001        3221        1559        1
+sigma                   0.511  0.037     0.438      0.575        0.001      0            2945        1522        1
+=====================  ======  =====  ========  =========  ===========  =========  ==========  ==========  =======
+
+.. code-block:: python
+
+   # Simulate new data conditioned on inferred parameters
+   new_x_data = pm.draw(
+    pm.Normal.dist(shape=(3, 3)),
+    random_seed=seed,
+   )
+   new_coords = coords | {"trial": [0, 1, 2]}
+
+   with inference_model:
+      pm.set_data({"x": new_x_data}, coords=new_coords)
+      idata = pm.sample_posterior_predictive(
+         idata,
+         predictions=True,
+         extend_inferencedata=True,
+         random_seed=seed,
+      )
+
+   pm.stats.summary(idata.predictions, kind="stats")
```

Comment:
Shouldn't we show all the summaries outputs? Why only the first?

Reply:
I think for starters it's TMI and can scare people off. Convergence diagnostics is more advanced than what we want to demo here.

Reply:
Oh, I thought you meant more columns, but you meant more rows?

Reply:
I mean't every time we print pm.stats.summary, we should show. I already removed the extra convergence columns witch `kind="stats"`